docs/plugin-synopsis.asciidoc.erb

```diff
@@ -39,6 +39,15 @@ Available configuration options:
    else
      annotation += "|No"
    end
+   if section == "filter" || section == "output"
+     if config[:allow_dynamic] == true
+       annotation += "|Yes"
+     elsif config[:allow_dynamic] == false
+       annotation += "|No"
+     else
+       annotation += "|Unknown"
```

Comment:
There's only two possibilities (true or false) so an 'unknown' doesn't make sense to me.

We can simplify the annotation stuff with a ternary:

```
annotation += config[:allow_dynamic] ? "|Yes" : "|No"
```